modular_skyrat/code/game/objects/items/storage/MREs.dm

```diff
@@ -0,0 +1,98 @@
+/obj/item/storage/mre
+	name = "standard MRE"
+	desc = "A vacuum-sealed bag containing a day's worth of nutrients for a crewmember in strenuous situations. There is no visible expiration date on the package."
+	icon = 'modular_skyrat/icons/obj/MRE.dmi'
+	icon_state = "mre"
+
+/obj/item/storage/mre/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	var/static/list/can_hold = typecacheof(list(
+		/obj/item/reagent_containers/food,
+		/obj/item/reagent_containers/glass/beaker/waterbottle,
+		/obj/item/kitchen/fork,
+		/obj/item/reagent_containers/food/condiment/pack/))
+	STR.can_hold = can_hold
+
+/obj/item/storage/mre/PopulateContents()
+	new /obj/item/reagent_containers/food/snacks/soup/mre(src)
+	switch (rand(1,2))
+		if(1)
+			new /obj/item/reagent_containers/food/snacks/sosjerky/healthy/mre(src)
+		if(2)
+			new /obj/item/reagent_containers/food/snacks/no_raisin/healthy/mre(src)
```

Comment:
use pick() instead

Reply:
tried pick() and it didn't work, I showed you this earlier.